examples/cmd/encrypt.go

```diff
@@ -46,7 +46,7 @@ func encrypt(cmd *cobra.Command, args []string) error {
 	defer tdfFile.Close()
 
 	tdf, err := client.CreateTDF(tdfFile, strReader,
-		//sdk.WithDataAttributes("https://example.com/attr/attr1/value/value1"),
+		// sdk.WithDataAttributes("https://example.com/attr/attr1/value/value1"),
```

Comment:
fixture provisioning is broken so this doesn't work in ci https://github.com/opentdf/platform/actions/runs/9057319882/job/24881138541?pr=759